validphys2/src/validphys/loader.py

```diff
@@ -326,26 +331,66 @@ def available_theories(self):
 
     @property
     @functools.lru_cache()
-    def available_datasets(self):
+    def _available_old_datasets(self):
+        """Provide all available datasets
+        At the moment this means cominbing the new and olf format datasets
+        """
         data_str = "DATA_"
+        old_commondata_folder = self.commondata_folder.with_name("commondata")
         # We filter out the positivity sets here
         return {
             file.stem[len(data_str) :]
-            for file in self.commondata_folder.glob(f'{data_str}*.dat')
+            for file in old_commondata_folder.glob(f'{data_str}*.dat')
             if not file.stem.startswith((f"{data_str}POS", f"{data_str}INTEG"))
         }
 
+    @property
+    @functools.lru_cache()
+    def available_datasets(self):
+        """Provide all available datasets.
+        At the moment this only returns old datasets for which we have a translation available
+        """
+        skip = ("POS", "INTEG")
+        old_datasets = [i for i in legacy_to_new_mapping.keys() if not i.startswith(skip)]
```

Comment:
I guess this is no longer necessary?
```suggestion
        old_datasets = [i for i in legacy_to_new_mapping.keys()]
```